blog/src/components/Featured/CardWide.jsx

```diff
@@ -0,0 +1,65 @@
+import images from "../../utils/images";
+import "../../styles/Card.scss";
+import PropTypes from "prop-types";
+import { useState } from "react";
+import { Link } from 'react-router-dom';
+
+
+const CardWide = ( {blogItemData} ) => {
+    const MAX_LENGTH = 250;
+    const [featured,setFeatured]=useState(blogItemData);
+    const [summary,setSummary]=useState(blogItemData.content)
```

Comment:
It is unnecessary to use the `useState` hook here as these values are not changing. I highly recommend to use the blogItemData directly